### DISCUSSION TITLE
请问在华硕arm8的路由器不通过软件中心怎样安装Alist3.8呢？它要开root权限才能安装，而华硕路由器不给开root

### DISCUSSION THREAD

**Discussion by fordiy** [AlistGo/alist#3067] [General]
请问在华硕路由器arm8的路由器不通过软件中心怎样安装Alist3.8呢？它要开root权限才能安装，而华硕路由器不给开root

**Comment by NovaShen555** [comment]
能ssh吗

**Reply by M-CU** [reply]
华硕官方固件现在支持ssh登录了，但我还是不懂通过ssh怎么安装alist 。恳求华硕路由器ssh安装alist的相关教程

**Reply by NovaShen555** [reply]
在release找到符合架构版本的发行版，下到机子上，用./xxx 文件名来运行，若运行成功，就可以添加到systemctl中
